jackson2/src/test/java/org/jdbi/v3/jackson2/TestJackson2Plugin.java

```diff
@@ -13,167 +13,22 @@
-        h.execute("create table whozits (id serial primary key, whozit json not null)");
-        final JsonDao dao = h.attach(JsonDao.class);
-        dao.insert(new Whozit("yams", 42));
-        dao.insert(new Whozit("apples", 24));
-
-        assertThat(dao.select())
-            .extracting("food", "bitcoins")
-            .containsExactlyInAnyOrder(
-                    new Tuple("yams", 42),
-                    new Tuple("apples", 24));
-    }
-
-    @Test
-    public void testJsonNested() {
-        h.execute("create table beany (id serial primary key, nested1 json, nested2 json)");
-        assertThat(h.createUpdate("insert into beany(id, nested1, nested2) values (:id, :nested1, :nested2)")
-            .bindBean(new Beany(42, 64, "quux"))
-            .execute()).isEqualTo(1);
-
-        Beany beany = h.createQuery("select * from beany")
-            .mapToBean(Beany.class)
-            .findOnly();
-
-        assertThat(beany.getId()).isEqualTo(42);
-        assertThat(beany.getNested1().getA()).isEqualTo(64);
-        assertThat(beany.getNested2().getB()).isEqualTo("quux");
-    }
-
-    @Test
-    public void testNull() {
-        h.execute("create table whozits (id serial primary key, whozit json)");
-        final JsonDao dao = h.attach(JsonDao.class);
-        dao.insert(null);
-        assertThat(h.createQuery("select whozit from whozits")
-                .mapTo(String.class)
-                .findOnly())
-            .isNull();
-        assertThat(dao.select())
-            .containsExactly(new Whozit[] {null});
-    }
-
-    public static class Beany {
-        private int id;
-        private N1 nested1;
-        private N2 nested2;
-
-        public Beany() {}
-        Beany(int id, int a, String b) {
-            this.id = id;
-            this.nested1 = new N1(a);
-            this.nested2 = new N2(b);
-        }
-
-        public int getId() {
-            return id;
-        }
-
-        public void setId(int id) {
-            this.id = id;
-        }
-
-        @Json
-        public N1 getNested1() {
-            return nested1;
-        }
-
-        public void setNested1(N1 nested1) {
-            this.nested1 = nested1;
-        }
-
-        @Json
-        public N2 getNested2() {
-            return nested2;
-        }
-
-        public void setNested2(N2 nested2) {
-            this.nested2 = nested2;
-        }
-    }
-
-    public static class N1 {
-        private final int a;
-
-        @JsonCreator
-        public N1(@JsonProperty("a") int a) {
-            this.a = a;
-        }
-
-        public int getA() {
-            return a;
-        }
-    }
-
-    public static class N2 {
-        private final String b;
-
-        @JsonCreator
-        public N2(@JsonProperty("b") String b) {
-            this.b = b;
-        }
-
-        public String getB() {
-            return b;
-        }
+    public void before() {
+        jdbi = db.getJdbi().installPlugin(new Jackson2Plugin());
```

Comment:
This is where I wish I could do `jdbi = db.getJdbi().installPlugin(new Jackson2Plugin(new ObjectMapper().registerModule(new ParameterNamesModule())))` but I guess splitting it across two lines isn't so bad.

Reply:
I won't stop you if you want to add it then